src/components/Card/index.js

```diff
@@ -0,0 +1,66 @@
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import Text from '../Text';
+
+const styles = {
+  padding({ size }) {
+    if (size === 's') return 'var(--f-space--xl)';
+    if (size === 'm') return 'var(--f-space--xxl)';
+
+    return undefined;
+  },
+  boxShadow({ isSelected, elevation }) {
+    if (elevation === 2) return '0 4px 20px rgba(25, 1, 52, 0.14)';
```

Comment:
do we have those box shadows in the theme?

Reply:
nope but I'll add them!